src/python/pants/backend/python/subsystems/python_tool_base.py

```diff
@@ -15,6 +15,9 @@
 from pants.option.subsystem import Subsystem
 from pants.util.ordered_set import FrozenOrderedSet
 
+DEFAULT_LOCKFILE = "<default>"
```

Comment:
I think these should include TOOL in the name. The concept is only relevant there. We can't provide a default lockfile for user requirements, and none is set differently.